Dockerfile

```diff
@@ -10,4 +10,7 @@ RUN pip install -r /source/requirements.txt
 
 RUN python setup.py develop
 
+RUN apt-get -y update
+RUN apt-get -y install vim
```

Comment:
I don't mind adding that for now but I think the direction should be to make the container lighter
I was thinking about moving to alpine with is much smaller 

BTW, looks like the image`python:3.6.5` is not available in Docker hub, the might might fail because of that
we should use `python:3.6` instead